solutions/usaco-1061.mdx

```diff
@@ -0,0 +1,109 @@
+---
+id: usaco-1061
+source: USACO Bronze 2020 December
+title: Stuck in a Rut
+author: Ryan Chou
+---
+
+[Official Analysis (C++ and Java)](http://www.usaco.org/current/data/sol_prob3_bronze_dec20.html)
+
+## Explanation
```

Comment:
```suggestion
## Solution
```

Reply:
i think this can be an optional change, its just that the rest of them say solution